### ISSUE TITLE
Forum for translators of QON wanted

### ISSUE THREAD

**Issue by Fabijenna** [pbek/QOwnNotes#2311]
#### Expected behaviour

FabijennAmy.Raw, [25.11.21 08:28]
I would like to have a Telegram group for translators. There are many questions and the Crowdin CAT tool doesn't support much exchange among the translators. To answer questions and point to the answers in the documentation but also solutions for technical or linguistic issues. Should I open a question on Github? my question is, where to announce that group so that all translators see the invitation to join. (Best would be a channel for announcements, and a corresponding group for discussions.) 😊 Probably the translation page in the documentation would be the best place. Agree?

FabijennAmy.Raw, [25.11.21 08:36]
or could we extend the page qownnotes / translation with a discussion page, a forum? might be better, as not everyone has TG

#### Actual behaviour

On Crowdin I can't suggest translation solutions to problems ppl are asking, and as I am never notified when someone asks a question on the translation on Crowdin, many questions left unanswered. I would also like to discuss problems and help each other for a better translation experience and outcome.

#### Output from the debug section in the settings dialog

<details><summary>Expand</summary>

</details>

#### Relevant log output in the Log panel

<details><summary>Expand</summary>

</details>

**Comment by pbek** [comment]
> announcing to all translators

I can only send a message on https://crowdin.com/project/qownnotes/members where I select all people. But all people would be in that chat. 

> where to announce that group so that all translators see the invitation to join. 

I haven't seen that.

> telegram

I don't think everyone is or wants to be on telegram.. E.g. Asians (especially Chinese) mostly use WeChat...
It might be hard to get everyone to a 2nd platform... 🤔

**Comment by pbek** [comment]
There's no setting I could find for an announcement.

**Comment by Fabijenna** [comment]
> I can only send a message on https://crowdin.com/project/qownnotes/members where I select all people. But all people would be in that chat.

Oh, that's new to me. If that opens a group chat, where everyone has access to, then I don't need anything else.
So far I have added people individually but Crowdin limits this way by 20 people and I didn't found that practical to use.

However I don't have access to the result of the above link, so please check whether you can change anything that works out for me. E.g. address everyone including me, and let's see if I can use it for my **infos to all**.

**Comment by Fabijenna** [comment]
![Uploading Screenshot 2021-11-25 at 11.11.51.png…]()

**Comment by Fabijenna** [comment]
![Screenshot 2021-11-25 at 11 11 51](https://user-images.githubusercontent.com/17498644/143422428-54879f63-f7c5-4a5a-b713-1bb92db9056f.png)

**Comment by Fabijenna** [comment]
I would like to have a space where everything is about QON translation and not coding or anything else, to focus on the best outcomes for translation.

**Comment by Fabijenna** [comment]
I want to scutinize the German translation and make a good example of it in what way we need to translate the documentation

**Comment by Fabijenna** [comment]
Here I suggest that we take these purposes to Crowdin at the keyword "QOwnNotes"
https://crowdin.com/translate/qownnotes/24/en-de?filter=basic&value=0#q=QOwnNotes

**Comment by Fabijenna** [comment]
As I don't want to make the Translation page in Documentation too crowded, what do you think of this option to detailed info and a changeable announcement of the answers we've been usually asked?

Same question, an other solution suggested, do you still need me to open another issue?

**Comment by pbek** [comment]
> As I don't want to make the Translation page in Documentation too crowded, what do you think of this option to detailed info and a changeable announcement of the answers we've been usually asked?

What option? Writing comments on a translation string? Do you think this would read anyone and how would it be found?

**Comment by Fabijenna** [comment]
I'm trying to be constructive here.
Maybe in the documentation but it could also be the documentation if you didn't mind.

**Comment by Fabijenna** [comment]
Point 10! 😱

**Comment by pbek** [comment]
All your "points" on https://www.qownnotes.org/contributing/translation.html are valid documentation in my eyes... On how translation should be conducted for the QOwnNotes project, along with hints and special cases. In my eyes these are no announcements... What do you want to announce? I think there is no other sensible way than Crowdin messages for announcements...

**Comment by Fabijenna** [comment]
Then please note my question above, where I have offered that you create the space for such communication and I'd manage to share the content in question, please.

**Comment by Fabijenna** [comment]
Brb

**Comment by Fabijenna** [comment]
Maybe "announcement" is the wrong word.
I keep seeing a need for improvement when I go over some of the work, that translators do, and I needed approximately 4 entire workdays just to go through German. Nothing special, but stuff that I would not like to have on my website, for real. It really needs to unravel at some points and that's nothing serious, it just needs time. 

Reasons that I want to send out a little bit more information for the translators, about the subjects that you too recognized. 

* Translators need to be aware of these issues BEFORE starting work on QON. It may be a part of the documentation, or any elsewhere (let us skip this question until we have all reasons), but people are asking in Crowdin the same questions, and mostly they don't even get answered. The translators are left in the dark about their questions while they have plausible answers, I could give when someone would just notify me to be able to answer. 
* Secondly I also don't think all translators see these communications because Crowdin is not working that way, it's chaotic. Mostly I read them only by chance. This point is relevant to quality and also a waste of resources.

I think I'd have some more to list here, but I would leave it at these two points to clarify.

What do you think about my reasons? Negligible or not?

**Comment by Fabijenna** [comment]
I could also live with another way to improve quality until there is a good option for it. You tell me.

**Comment by Fabijenna** [comment]
Give me some time, then I figure it out.

**Comment by Fabijenna** [comment]
Right. I see what you meant. Please tell me how I could be of support. How can I help?
